Source/Particles/ElementaryProcess/Make.package

```diff
@@ -1,7 +1,10 @@
 ifeq ($(QED),TRUE)
+<<<<<<< HEAD
```

Comment:
careful: you committed an unresolved merge conflict :)

Reply:
Thanks. :) Though they should be resolved in the last commit no?

Reply:
Gone now, maybe I commented on an outdated commit.

Reply:
Gone now, maybe I commented on an outdated commit.